### PR TITLE
[UXE-3942] test: e2e domains major cases

### PR DESCRIPTION
**Pull request by peterpaulo-azion**
### What is the new behavior introduced by this PR?
* test options and edit of domains
* test trustedCa digital certificate

### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does this PR introduce UI changes? Add a video or screenshots here.
no

### Does it have a link on Figma?
no
<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [x] Edge
- [x] Firefox
- [ ] Safari

### CODE REVIEW

File: cypress/support/selectors.js [hunk 1]
```diff
@@ -28,6 +29,14 @@ const selectors = {
...
   },
   form: {
+    teamName: '[data-testid="teams-permissions-form__name__field-text__input"]',
+    teamNameError: '[data-testid="teams-permissions-form__name__field-text__error-message"]',
+    teamStatus: '[data-testid="teams-permissions-form__form-fields__status"]',
+    allPermissionsToTarget: '[aria-label="Move All to Target"] > .p-icon',
+    allPermissionsToSource: '[aria-label="Move All to Source"] > .p-icon',
+    singlePermissionToTarget: 'button[aria-label="Move to Target"]',
+    viewContentDeliverySettingsPermission:
+      '[data-testid="teams-permissions-form__permissions-field__picklist__item-View Content Delivery Settings"]',
```
Comment (william-tome) [inline cypress/support/selectors.js:39]: Foram migrados para a seção de teams (linha 432)
```suggestion
```

File: cypress/support/selectors.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 const selectors = {
   list: {
+    createTeamButton: '[data-testid="create_Team_button"]',
```
Comment (william-tome) [inline cypress/support/selectors.js:3]: Migrado para a seção de teams (linha 432)
```suggestion
```

File: cypress/e2e/domains.cy.js [hunk 3]
```diff
@@ -47,6 +165,23 @@ describe('Domains spec', () => {
       'Succesfully created!',
       'The domain is now available in the Domain management section.'
     )
+
+    // Act
+    cy.get(selectors.domains.fieldTextInput).clear('', { delay: 3000 })
```
Comment (william-tome) [inline cypress/e2e/domains.cy.js:170]: Por que esse delay?

File: cypress/e2e/digital-certificates.cy.js [hunk 4]
```diff
@@ -11,23 +12,45 @@ describe('Digital Certificates spec', () => {
...
   })
 
+  it('should create a truted CA digital certificate', function () {
+    // Arrange
+    cy.get(selectors.digitalCertificates.createDigitalCertificateButton).click()
+    cy.get(selectors.digitalCertificates.digitalCertificateName).clear()
+    cy.get(selectors.digitalCertificates.digitalCertificateName).type(digitalCertificateName)
+    cy.get(selectors.digitalCertificates.importTrustedCARadioOption).click()
+    cy.get(selectors.digitalCertificates.trustedCATextArea).type(fixtures.templateCertificate, {
+      delay: 0
```
Comment (william-tome) [inline cypress/e2e/digital-certificates.cy.js:39]: Por que esse delay?

File: cypress/e2e/domains.cy.js (hunk 3 above)
Comment (peterpaulo-azion) [reply cypress/e2e/domains.cy.js]: Esse foi removido.

File: cypress/e2e/digital-certificates.cy.js (hunk 4 above)
Comment (peterpaulo-azion) [reply cypress/e2e/digital-certificates.cy.js:39]: Como a string é muuito longa, ele estava ultrapassando o timeout e falhando no teste. O delay 0 faz com que a string seja inserida de uma vez, tal como um Command + V.

Comment (william-tome) [reply cypress/e2e/digital-certificates.cy.js:39]: Vivendo e aprendendo.

File: cypress/e2e/domains.cy.js [hunk 5]
```diff
@@ -1,29 +1,147 @@
...
+    cy.verifyToast('Successfully copied!')
+    cy.get(selectors.domains.confirmButton).click()
+    cy.get(selectors.domains.editPageTitle).should('have.text', 'Edit Domain')
+    cy.get(selectors.domains.mtlsTrustedCAFieldSelectedValue).should(
+      'contain',
+      digitalCertificateName
+    )
+  })
+
+  it.only('should edit a domain successfully', () => {
```
Comment (william-tome) [inline cypress/e2e/domains.cy.js:142]:
```suggestion
  it('should edit a domain successfully', () => {
```

**Review by william-tome — APPROVED** [review]

**Review by HerbertJulio — APPROVED** [review]

**Pull request merged**